### PR TITLE
Update el.json

### PR DESCRIPTION
**Pull request by arius28**

### CODE REVIEW

File: lang/el.json [hunk 1]
```diff
@@ -108,17 +108,17 @@
 	"ErrorUnknown": "Η ενέργεια δεν ήταν δυνατόν να εκτελεστεί. (Σφάλμα %1)",
 	"Errors": {
 		"10": "Λανθασμένη Εντολή.",
-		"11": "Το resource type δεν ήταν δυνατόν να προσδιοριστεί.",
-		"12": "Το resource type δεν είναι έγκυρο.",
-		"13": "The connector configuration file is invalid.",
-		"14": "Invalid connector plugin: %1.",
+		"11": "Ο τύπος πόρου(resource type) δεν ήταν δυνατόν να προσδιοριστεί.",
+		"12": "Ο τύπος πόρου(resource type) δεν είναι έγκυρος.",
+		"13": "Το αρχείο ρυθμίσεων του πρόσθετου(plugin) σύνδεσης είναι άκυρο.",
```
Comment (AnnaTomanek) [inline lang/el.json]: To make things more clear here and below: the "connector" is simply the server-side part of CKFinder, it's not an additional plugin.

File: lang/el.json [hunk 2]
```diff
@@ -108,17 +108,17 @@
...
+		"14": "Άκυρο πρόσθετο(plugin) σύνδεσης: %1.",
```
Comment (AnnaTomanek) [inline lang/el.json:114]: So this message means that one of the server-side CKFinder plugins is invalid/incorrect.

Comment (arius28) [inline lang/el.json:114]: Hi Anna,

I am a bit confused. You mean that the connector should not be regarded as a plugin, so I should change my translation to reflect that OR should it be regarded as a plugin?

Aris

Comment (AnnaTomanek) [inline lang/el.json:114]: Hi Aris,

Answer 1: connector is not a plugin. CKFinder consists of 2 parts: client-side part written in JavaScript and server-side part (also known as the "connector", currently in PHP). CKFinder plugins come in two types: the client-side plugins and the server-side plugins. This message is displayed when one of the server-side ("connector") plugins is incorrect.

Hope this helps!

**Comment by AnnaTomanek** [comment]
Hi Aris, thanks a lot for your contribution! Your pull request has just been merged and your changes will be available in the next release.

A minor note, though: I corrected some missing spaces and parentheses in commit de3616f1f62c136bc6011a29f0adfb40159e988c.

By the way, if you were willing to add translations for the Image Edit presets, you can find samples of what they do on the CamanJS [Interactive Examples](http://camanjs.com/examples/) page.

As promised, we have a free CKFinder 3 license for anyone that provides a significant translation update. If you ever need such a license, just contact us and send the URL that your CKFinder installation will be used at and we'll provide you the license key.

Thanks again for your help, it is much appreciated both by development team and CKFinder users!

**Comment by arius28** [comment]
Thank you Anna!

The minor corrections were good, thank you for that also. 
I did spot a small spelling error on my part. It is really small. Should I send another pull request or can you make the correction on the spot?
It is for line 213:
"max": "Μέγιστπ",
should be
"max": "Μέγιστο",

Regarding the Image edit presets I 'll give it a go soon.

On a side note. Some of the filters were not available in the public demo I saw of CKFinder 3, for example clip, noise etc. Is there a demo where I can see them? 

Finally, thank you for the CKFinder 3 license bonus!

**Comment by AnnaTomanek** [comment]
Hi Aris,

I fixed line 213 in 5cc1e72f4a5809c794de2d7d9b78c2fba5080191. Thanks for spotting this!

As for image editing features, they come from an external library, CamanJS. There is a nice [Interactive Examples](http://camanjs.com/examples/) page there with some samples, should be helpful. Another idea is to check your favorite (localized) photo editing application for clues, you can try Gimp, Paint.NET etc.

Thanks you for helping out!

**Pull request merged**